elide-spring/elide-spring-boot-starter/README.md

```diff
@@ -0,0 +1,24 @@
+# Elide Spring Boot Starter
+
+Opinionated jar which packages dependencies to get started with Elide and Spring Boot.  The starter includes:
+1. Spring Boot Web (minus tomcat)
+2. Spring Boot Jetty
+3. Spring Boot JPA
+4. YAML Configuration
+5. All of the Elide dependencies
+
+## Maven Dependency
+
+```xml
+   <dependency>
+       <groupId>com.yahoo.elide</groupId>
+       <artifactId>elide-spring-boot-starter</artifactId>
+       <version>${elide.version}</version>
+   </dependency>
+```
+
+## Example Usage.
+
+An example project can be viewed [here](https://github.com/aklish/elide-spring).
```

Comment:
Your example repo contains this, but it might be worth describing how override the default application.yaml configuration that is bundled with elide-spring. Particularly since it seems likely that different configuration will be desirable once the example model is up and running.